eng/pipelines/templates/jobs/archetype-sdk-client.yml

```diff
@@ -111,7 +111,15 @@ jobs:
     pool:
       vmImage: '$(OSVmImage)'
 
-    steps:    
+    steps:
```

Comment:
Why do we need this here as well?

Reply:
Yes, so that the artifacts published has the right version.